src/app/users/users.component.ts

```diff
@@ -157,27 +157,17 @@ export class UsersComponent implements OnInit, AfterViewInit {
     const myUserId = '_users/org.couchdb.user:' + this.userService.get().name;
     this.couchService.allDocs('shelf').subscribe(shelfUser => {
       shelfUser.map(shelf => {
-        const myTeamIds = [ ...shelf.myTeamIds ] || [];
-        if(myTeamIds.indexOf(idShouldBeRemove)) {
+        if ('myTeamIds' in shelf) {
+          const myTeamIds = [ ...shelf.myTeamIds ] || [];

```

Comment:
this code is supposed to set empty array if there is no myTeamIds so if condition should not be needed